src/NzbDrone.Core.Test/MediaFiles/EpisodeImport/ImportDecisionMakerFixture.cs

```diff
@@ -55,6 +56,16 @@ public void Setup()
 
             _series = Builder<Series>.CreateNew()
                                      .With(e => e.Profile = new Profile { Items = Qualities.QualityFixture.GetDefaultQualities() })
+                                     .With(l => l.LanguageProfile =
+                                         new LanguageProfile
+                                         {
+                                             Languages = new List<ProfileLanguageItem>
```

Comment:
Same here, we're creating a new Language profile multiple times, but for quality profile we just call:

`.With(e => e.Profile = new Profile { Items = Qualities.QualityFixture.GetDefaultQualities() })`